src/o365/spfx/commands/project/project-upgrade.ts

```diff
@@ -488,6 +507,78 @@ ${f.resolution}
     return s.join('').trim();
   }
 
+  private getTourReport(findings: FindingToReport[]): string {
+    const tourFindings: FindingTour = {
+      title: "Upgrade Project",
+      steps: []
+    };
+
+    findings.forEach(f => {
+
+      const lineNumber: number = f.position && f.position.line ? f.position.line : this.getLineToModify(f);
+      
+      let resolution: string = '';
+      switch (f.resolutionType) {
+        case 'cmd':
+          resolution = `Execute the following command:\r\n>> ${f.resolution}`;
+          break;
+        case 'json':
+        case 'js':
+        case 'ts':
+        case 'scss':
+          resolution = `Update the code as follows:\r\n\`\`\`${f.resolutionType}\r\n${f.resolution}\r\n\`\`\``;
```

Comment:
This message seems confusing:

<img width="688" alt="image" src="https://user-images.githubusercontent.com/11164679/82242131-a0ea8400-993d-11ea-9941-167333c3a4a9.png">

We first say to remove the line and then to update it. Since the resolution already contains text, shouldn't we use it and skip the fixed `Update the code as follows:` string?